From cd2fec6ca42702e81729492b222d4b1ed4c78bea Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=89=8B=E7=93=9C=E4=B8=80=E5=8D=81=E9=9B=AA?=
 <nanaeonn@outlook.com>
Date: Thu, 23 Oct 2025 14:19:25 +0800
Subject: [PATCH 6/7] Add SILK demuxer and muxer to Makefile

Registered silkdec.o and silkenc.o for SILK demuxer and muxer in libavformat/Makefile to enable support for SILK format.
---
 libavformat/Makefile                     | 2 ++
 libavformat/{silkdec.c => silkdemuxer.c} | 0
 libavformat/{silkenc.c => silkmuxer.c}   | 0
 3 files changed, 2 insertions(+)
 rename libavformat/{silkdec.c => silkdemuxer.c} (100%)
 rename libavformat/{silkenc.c => silkmuxer.c} (100%)

diff --git a/libavformat/Makefile b/libavformat/Makefile
index 7ca68a7036..41ff5b8ab7 100644
--- a/libavformat/Makefile
+++ b/libavformat/Makefile
@@ -222,6 +222,8 @@ OBJS-$(CONFIG_FLIC_DEMUXER)              += flic.o
 OBJS-$(CONFIG_FLV_DEMUXER)               += flvdec.o
 OBJS-$(CONFIG_LIVE_FLV_DEMUXER)          += flvdec.o
 OBJS-$(CONFIG_FLV_MUXER)                 += flvenc.o
+OBJS-$(CONFIG_SILK_DEMUXER)              += silkdemuxer.o
+OBJS-$(CONFIG_SILK_MUXER)                += silkmuxer.o
 OBJS-$(CONFIG_FOURXM_DEMUXER)            += 4xm.o
 OBJS-$(CONFIG_FRAMECRC_MUXER)            += framecrcenc.o framehash.o
 OBJS-$(CONFIG_FRAMEHASH_MUXER)           += hashenc.o framehash.o
diff --git a/libavformat/silkdec.c b/libavformat/silkdemuxer.c
similarity index 100%
rename from libavformat/silkdec.c
rename to libavformat/silkdemuxer.c
diff --git a/libavformat/silkenc.c b/libavformat/silkmuxer.c
similarity index 100%
rename from libavformat/silkenc.c
rename to libavformat/silkmuxer.c
-- 
2.47.0.windows.2

